src/main/java/com/epam/ta/reportportal/database/entity/settings/SamlProviderDetails.java

```diff
@@ -0,0 +1,109 @@
+package com.epam.ta.reportportal.database.entity.settings;
+
+import java.io.Serializable;
+
```

Comment:
Could you please add the author and some java docs describing the class?

Reply:
Javadoc for the parameters have been added